testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/EnvironmentContainers.java

```diff
@@ -34,4 +38,23 @@ public static boolean isPrestoContainer(String name)
     {
         return name.startsWith(PRESTO);
     }
+
+    /**
+     * Use this method only when you place `tempto-configuration.yaml' in environment configuration directory.
+     */
+    public static void configureTempto(Environment.Builder builder, DockerFiles.ResourceProvider configDir)
+    {
+        builder.configureContainer(TESTS, dockerContainer -> {
+            String environmentName = configDir.getPath("..").toFile().getName();
```

Comment:
Can we try something like this 
`configDir.getPath("tempto-configuration.yaml").getParent().toFile().getName()`

Reply:
What is the difference?